src/tcia_utils/nbia.py

```diff
@@ -910,6 +928,153 @@ def getDoiMetadata(doi, output, api_url = "", format = ""):
         print(errt)
     except requests.exceptions.RequestException as err:
         print(err)
+
+####### getSimpleSearchCriteriaValues function
+# Takes the same parameters as the SimpleSearch GUI 
+# Using more parameters narrows the values received. 
+def getSimpleSearchCriteriaValues(
+    collections = [],
+    species = [],
+    modalities = [],
+    bodyParts = [],
+    manufacturers  = [],
+    fromDate = "",
+    toDate = "",
+    patients = [],
+    minStudies: int = 1,
+    modalityAnded = False,
+    api_url = "",
+    format = ""):
+    """
+    Takes the same parameters as the SimpleSearch GUI; using more parameters narrows 
+    the values received. 
+
+    collections: list[str]   -- The DICOM collections of interest to you
+    species: list[str]       -- Filter collections by species. Possible values are 'human', 'mouse', and 'dog' 
+    modalities: list[str]    -- Filter collections by modality
+    modalityAnded: bool      -- If true, only return collections with all requested modalities, as opposed to any
+    minStudies: int          -- The minimum number of studies a collection must have to be included in the results
+    manufacturers: list[str] -- Imaging device manufacturers, e.g. SIEMENS
+    bodyParts: list[str]     -- Body parts of interest, e.g. CHEST, ABDOMEN
+    fromDate: str            -- First cutoff date, in DD/MM/YYYY format. Defaults to 1 January 1900
```

Comment:
I'm confused by this request. If I go to the SimpleSearch webpage, open developer tools, go to the network tab, enter in a date range, then apply it, then I can see a call made to getSimpleSearchWithModalityAndBodyPartPaged (gSSWMABPP) with the dates in DD/MM/YYYY format. 

I certainly can change this code so that the user enters dates in YYYY/MM/DD format, but the other way around is what I see your web GUI using, and for an endpoint that isn't the odd one out you specified. Is this conversion happening on the server side for gSSWMABPP?

Reply:
Well actually in the GUI widget it's MM/DD/YYYY, it's the request that's DD/MM/YYYY.

Reply:
Interesting, there is even more variability in handling dates across the different parts of NBIA than I realized.  But to clarify the reason for my request, when I started working on tcia-utils the first date-related inputs I encountered were https://wiki.cancerimagingarchive.net/display/Public/NBIA+Search+REST+API+Guide#NBIASearchRESTAPIGuide-NewPatientsInCollection and https://wiki.cancerimagingarchive.net/display/Public/NBIA+Search+REST+API+Guide#NBIASearchRESTAPIGuide-NewStudiesInPatientCollection which both used the YYYY/MM/DD.  Then later I went to add support for https://wiki.cancerimagingarchive.net/display/Public/NBIA+Search+REST+API+Guide#NBIASearchRESTAPIGuide-getUpdatedSeries but noticed the API expects MM/DD/YYYY for that endpoint, but decided to do that conversion behind the scenes so that all date-related inputs for tcia-utils would be YYYY/MM/DD.  That's the only reason I'm suggesting we continue to stick with that format for getSimpleSearchCriteriaValues.

Reply:
OK, I'm not attached to either format, I just wanted to ensure if we changed it we'd actually be changing it to the consistent one. :) I'll push in a sec.